enterprise_access/apps/subsidy_request/management/commands/send_admins_email_with_new_requests.py

```diff
@@ -0,0 +1,44 @@
+"""
+Management command to send admins email with new subsidy requests.
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+
+from enterprise_access.apps.subsidy_request.models import SubsidyRequestCustomerConfiguration
+from enterprise_access.apps.subsidy_request.tasks import send_admins_email_with_new_requests_task
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    This command is intended to run as frequently as specified in a crontab. The window of time used
+    to grab requests that were made will be from the time this is last run (stored in the
```

Comment:
Opening ( without a )

Reply:
Could you restate this for clarity: 

> The window of time used  to grab requests that were made will be from the time this is last run 

Do you mean that the command usees the last _updated_field on SubsidyRequestCustomerConfiguration model to determine the window for the job run?

Reply:
I think yes, but lemme say again:

We plan to run the job every 12 hours, which is going to be defined in configuration/crontab elsewhere.

The email that gets sent out will only include subsidy requests created later than or equal to the last time this job ran. And the timestamp for when it was last run is on the `last_remind_date` on the SubsidyRequestCustomerConfiguration model

Reply:
reworded!